tests/e2e/machineregistration_test.go

```diff
@@ -23,54 +23,60 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	http "github.com/rancher-sandbox/ele-testhelpers/http"
-	elementalv1 "github.com/rancher/elemental-operator/pkg/apis/elemental.cattle.io/v1beta1"
+
 	"github.com/rancher/elemental-operator/pkg/config"
-	corev1 "k8s.io/api/core/v1"
-	rbacv1 "k8s.io/api/rbac/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	"github.com/rancher/elemental-operator/tests/catalog"
+
+	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
 )
 
 var _ = Describe("MachineRegistration e2e tests", func() {
 	Context("registration", func() {
-		var mRegistration *elementalv1.MachineRegistration
+		var mRegistration string = "machine-registration"
+		k := kubectl.New()
 
 		BeforeEach(func() {
-			mRegistration = &elementalv1.MachineRegistration{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "machine-registration",
-					Namespace: operatorNamespace,
-				},
-			}
+			var mRegistration string = "machine-registration"
+			mr := catalog.NewMachineRegistration(
+				mRegistration,
+				&config.Config{},
+			)
+
+			Eventually(func() error {
+				return k.ApplyYAML(operatorNamespace, mRegistration, mr)
+			}, 1*time.Minute, 2*time.Second).ShouldNot(HaveOccurred())
+
 		})
 
 		AfterEach(func() {
-			Expect(cl.Delete(ctx, mRegistration)).To(Succeed())
-
 			Eventually(func() bool {
-				if err := cl.Get(ctx, client.ObjectKeyFromObject(mRegistration), &rbacv1.Role{}); !apierrors.IsNotFound(err) {
-					return false
-				}
-				if err := cl.Get(ctx, client.ObjectKeyFromObject(mRegistration), &corev1.ServiceAccount{}); !apierrors.IsNotFound(err) {
-					return false
-				}
-				if err := cl.Get(ctx, client.ObjectKeyFromObject(mRegistration), &rbacv1.RoleBinding{}); !apierrors.IsNotFound(err) {
-					return false
+				machineTypes := [3]string{"Role", "ServiceAccount", "RoleBinding"}
+
+				for i := 0; i < len(machineTypes); i++ {
+					r, err := kubectl.GetData(operatorNamespace, machineTypes[i], mRegistration, `jsonpath={}`)
```

Comment:
same question as above, what is the reason for removing controller-runtime stuff?